src/main/resources/templates/bo/pages/promotion.html

```
@@ -2,46 +2,383 @@
+    </div>
+
+    <!-- ===== Discount Code Tab ===== -->
+    <div id="promo-section-discount" class="space-y-6">
+
+        <!-- Create Form -->
+        <div class="bg-white rounded-xl shadow-sm border border-slate-200 p-6">
+            <h3 class="font-bold text-slate-700 mb-4">할인 코드 생성</h3>
+            <div class="flex flex-wrap gap-4 items-end">
+                <div class="flex flex-col gap-1">
+                    <label class="text-xs font-medium text-slate-500">학회/기관명 <span class="text-red-500">*</span></label>
+                    <input type="text" id="discount-org-name" placeholder="예: 한국컴퓨터학회"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)